src/main/scala/org/hammerlab/guacamole/commands/VAFHistogram.scala

```diff
@@ -182,14 +181,15 @@ object VAFHistogram {
         sc.parallelize(histogramOutput).saveAsTextFile(args.output)
       } else {
         // Print histograms to standard out
-        variantAlleleHistograms.foreach(histogram =>
+        for {
+          (_, histogram) <- variantAlleleHistograms
+        } {
           histogram.toSeq.sortBy(_._1).foreach(kv => println(histogramToString(kv)))
-        )
+        }
       }
 
       if (args.cluster) {
-        val numClusters = args.numClusters
-        variantLoci.foreach(buildMixtureModel(_, numClusters))
+        buildMixtureModel(variantLoci, args.numClusters)
```

Comment:
I don't think I get this completely. It seems like a lot of the code changes here are to more efficiently handle multiple inputs, which seems great, but then this part produces a different result than it did before.  It seems either this should support multiple samples (and then give multiple models) or just a single sample. But now it seems structured to take multiple samples and produce a single model, which wouldn't be a relevant use case?  

It just seems like something different then it did before, defeats the purpose of making it more efficient?